news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,82 @@
+- [Remove unnecessary scrollbar on customization fields on new order page](https://github.com/PrestaShop/PrestaShop/issues/26535)
+- [Remove lazy load for logo on Front Office](https://github.com/PrestaShop/PrestaShop/issues/26666)
+- [Replaces the new product flag in the product flags by new](https://github.com/PrestaShop/PrestaShop/issues/26640)
+- [Fix listing classes when using facetedsearch](https://github.com/PrestaShop/PrestaShop/issues/26820)
+- [Fix dropdown align on email theme page](https://github.com/PrestaShop/PrestaShop/issues/26604)
+- [Fix width and height value for product image in listing template](https://github.com/PrestaShop/PrestaShop/issues/25849)
+- [Remove header calculation and set toolbar to sticky](https://github.com/PrestaShop/PrestaShop/issues/26383)
+- [Fix a bad alignment of label in the email theme page](https://github.com/PrestaShop/PrestaShop/issues/26915)
+- [Fix the newsletter block overlap on customer reassurance  block](https://github.com/PrestaShop/PrestaShop/issues/26846)
+- [CI is broken on autoupgrade - PHPStan with 1.6.1.18](https://github.com/PrestaShop/PrestaShop/issues/26861)
+- [Remove duplication of cache folders in letting Symfony handle kernel reboot for tests](https://github.com/PrestaShop/PrestaShop/issues/26924)
+- [Fix untranslatable Tabs for shops upgraded to 1.7.8.0 (BC Break)](https://github.com/PrestaShop/PrestaShop/issues/26933)
+- [Fix menu and searchbar on tablet on classic (BC Break)](https://github.com/PrestaShop/PrestaShop/issues/26643)
+- [Allow vertical scroll on product images](https://github.com/PrestaShop/PrestaShop/issues/26597)
+- [Fix product width on featuredproducts](https://github.com/PrestaShop/PrestaShop/issues/26992)
+- [Allow all characters for Cart Rule & Order Cart Rule names](https://github.com/PrestaShop/PrestaShop/issues/25829)
+- [Add an option to regenerate RTL file during the upgrade](https://github.com/PrestaShop/PrestaShop/issues/27307)
+
+
+## Modules 
+
+- [Excludes vendor, lib, tests directories when scanning modules' directories to extract translation catalogue](https://github.com/PrestaShop/PrestaShop/issues/24987)
+- [Fix color picker bug when multiple colorpickers are present](https://github.com/PrestaShop/PrestaShop/issues/25013)
+- [Error with dynamical form rendering in 1.7.8.1](https://github.com/PrestaShop/PrestaShop/issues/26786)
+- [Fix product width on featured products](https://github.com/PrestaShop/PrestaShop/issues/26705)
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.3).
+
+## Security fix 
+
+One security fix have been included in this patch version:
+
+- Server Side Twig Template Injection [security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-mrq4-7ch7-2465)
+
+More information about why it is important to update:
+
+## Acknowledgments
+
+Contributors to this patch version, from both the Core team and the community at large:
```

Comment:
I think we were supposed to avoid using "Core team" as it's the name of the team within the company. Maybe we could say "maintainers", even though it's not always quite right.

Reply:
```suggestion
Contributors to this patch version:
```
Simple